src/main/java/io/kestra/plugin/notion/CreatePage.java

```diff
@@ -0,0 +1,277 @@
+    description = "Creates a new page in Notion with the specified title and markdown content. Optionally specify a parent page for hierarchical organization."
+)
+@Plugin(
+    examples = {
+        @Example(
+            title = "Create a simple page",
+            full = true,
+            code = """
+                id: notion_create_page
+                namespace: company.team
+
+                tasks:
+                  - id: create_page
+                    type: io.kestra.plugin.notion.CreatePage
+                    apiToken: "{{ secret('NOTION_API_TOKEN') }}"
+                    title: "Meeting Notes"
+                    content: |
+                      # Meeting Notes - {{ now() }}
+                      
+                      ## Attendees
+                      - John Doe
+                      - Jane Smith
+                      
+                      ## Action Items
+                      - [ ] Review proposal
+                      - [ ] Schedule follow-up
+                """
+        ),
+        @Example(
+            title = "Create a page with parent",
+            full = true,
+            code = """
+                id: notion_create_child_page
+                namespace: company.team
+
+                tasks:
+                  - id: create_child_page
+                    type: io.kestra.plugin.notion.CreatePage
+                    apiToken: "{{ secret('NOTION_API_TOKEN') }}"
+                    title: "Sprint Planning"
+                    parentPageId: "12345678-1234-1234-1234-123456789abc"
+                    content: |
+                      # Sprint Planning
+                      
+                      **Sprint Goal:** Improve user authentication
+                      
+                      ## Stories
+                      - Implement OAuth2
+                      - Add password reset
+                """
+        )
+    }
+)
+public class CreatePage extends NotionConnection implements RunnableTask<CreatePage.Output> {
+
+    @Schema(
+        title = "Page title",
+        description = "The title of the new page"
+    )
+    @NotNull
+    private Property<String> title;
+
+    @Schema(
+        title = "Page content",
+        description = "The content of the page in markdown format"
+    )
+    private Property<String> content;
+
+    @Schema(
+        title = "Parent page ID",
+        description = "Optional parent page ID for creating a child page. If not specified, the page will be created at the workspace root."
+    )
+    private Property<String> parentPageId;
+
+    @Override
+    public Output run(RunContext runContext) throws Exception {
+        Logger logger = runContext.logger();
+
+        // Render and validate inputs
+        String renderedTitle = runContext.render(this.title).as(String.class).orElseThrow(
+            () -> new IllegalArgumentException("title is required")
+        );
+
+        String renderedContent = runContext.render(this.content).as(String.class).orElse("");
```

Comment:
If ` null` is supported better use it.
Otherwize this may just be a mandatory parameter